### ISSUE TITLE
add method dispose to be more eco friendly

### ISSUE THREAD

**Issue by VirusW** [stanislavT/atbmarket#13]
I'd like to use your package, but I have concerns about environment safety. Does it have a dispose method? How can I delete the object? Don't tell me about garbage collector! It doesn't sort my waste.

**Comment by stanislavt** [comment]
@VirusW yes, you are right. Let's add this method and make the environment safe.
